src/hivpy/sexual_behaviour.py

```diff
@@ -27,61 +26,78 @@ class FemaleSexBehaviour(IntEnum):
 class SexualBehaviourModule:
 
     def select_matrix(self, matrix_list):
-        return matrix_list[np.random.choice(matrix_list.shape[0])]
-
-    def _norm_probs(self, prob_dict: dict):
-        return {
-            key: data / sum(data)
-            for key, data in prob_dict.items()
-        }
+        return matrix_list[np.random.choice(len(matrix_list))]
```

Comment:
I may have suggested this before and forgotten the reply, sorry. 
```suggestion
        return np.random.choice(matrix_list)
```

Reply:
As above; causes error.